### ISSUE TITLE
TypeError: Cannot read property 'isProvided' of undefined

### ISSUE THREAD

**Issue by 19majkel94** [sharegate/craco#37]
After upgrade to 3.1.0, I've started to get this error when I'm running the tests.

```
FAIL src/Recordings/recordings.model.test.ts
  ● Test suite failed to run

    TypeError: Cannot read property 'isProvided' of undefined

      at log (node_modules/@craco/craco/lib/logger.js:4:17)
      at Object.<anonymous> (node_modules/@craco/craco/lib/paths.js:36:1)
      at Object.<anonymous> (node_modules/@craco/craco/lib/config.js:3:25)
```

Looking into the source code, you have circular dependencies between `loggger.js` and `args.js`, so one of them is resolved to an empty object, resulting in this error.

**Comment by 19majkel94** [comment]
@patricklafrance 
After commenting that line:
https://github.com/sharegate/craco/blob/3dfbc5089459f9a9af88e9cb8523e8d07655bf2f/packages/craco/lib/args.js#L37
my tests are passing, so it's clearly circular deps problem.
I can reproduce it also on v3.0.0.

**Comment by patricklafrance** [comment]
Hi @19majkel94 

Thanks for the issue.

You're right about that circular dependency. I don't understand why the tests in my local sandbox were running without that error.

Anyway, it's fix in 3.1.1

Thank you :)
